### PR TITLE
Add counties to territories, refs opendatateam/udata#466

### PR DESCRIPTION
**Pull request by davidbgk**
See https://github.com/opendatateam/udata/pull/467

### CODE REVIEW

File: udata_gouvfr/models.py [hunk 1]
```diff
@@ -32,164 +32,193 @@
...
-    title = 'Zonages des politiques de la ville'
-    # Ministère de l'Egalité des Territoires et du Logement.
-    organization_id = '534fff8fa3a7292c64a77f3d'
-    url_template = (
-        'http://sig.ville.gouv.fr/Territoire/{code}/onglet/DonneesLocales')
+class PopulationDataset(TerritoryDataset):
+    order = 1
+    title = 'Population'
+    # Institut National de la Statistique et des Etudes Economiques (INSEE).
+    organization_id = '534fff81a3a7292c64a77e5c'
```
Comment (noirbizarre) [inline udata_gouvfr/models.py]: We will need to find another pattern (later). Referencing and hard coding production identifier is not safe.

Comment (davidbgk) [inline udata_gouvfr/models.py]: Agreed 😓

File: udata_gouvfr/theme/theme.py [hunk 2]
```diff
@@ -33,6 +33,7 @@
         nav.Item(_('Organizations'), 'organizations.list'),
     ]),
     nav.Item(_('Dashboard'), 'site.dashboard'),
+    nav.Item(_('Territories'), 'territories.home'),
```
Comment (noirbizarre) [inline udata_gouvfr/theme/theme.py]: :+1:

**Comment by noirbizarre** [comment]
r+wc (but we need to have a discussion on the referencing pattern later)

**Comment by noirbizarre** [comment]
maybe try to pass the test when https://github.com/opendatateam/udata/pull/483 is merged

**Pull request merged**